viz_config.yml

```diff
@@ -12,6 +12,7 @@ precip_bin: 4 # inches per bin
 gage_flood_col: "#175676"
 gage_norm_col: "#4BA3C3"
 hurricane_col: "orange"
+n_intro_frames: 20 # number of frames in the intro animation linking gages to sparklines
 frame_delay_cs: 8 # delay between frames of the gif animation. cs = centiseconds = hundredths of a second
```

Comment:
...we may want a config var for the frame_delay for the intro frames...
:+1: to keep them the same for now, but just noting that.

Reply:
i'm open to that